src/css/components/panel.scss

```diff
@@ -27,6 +27,19 @@ $panel-padding: $grid-3;
     margin-bottom: $grid-1;
   }
 
+  .route-form {
+    background-color: $color-lightestgray;
+
+    label {
+      color: $color-blue;
+    }
+
+    .route-form-preview {
+      font-size: inherit;
+      padding-left: 0;
+    }
+  }
+
```

Comment:
Setting background color to match the other form backgrounds. Also updating the label color to use a darker blue as the previous label color on the light gray background did not meet accessibility for color contrast.

Also updating the font size of the URL preview to inherit.